app/src/context/DocumentStore.tsx

```diff
@@ -22,7 +22,7 @@ export const DocumentsContext = React.createContext<DocumentsValue | undefined>(
 
 export const DocumentProvider: React.FC<{
   children: React.ReactNode;
-}> = (props) => {
+}> = (props) => {    
   const [files, setFiles] = React.useState<File[]>([]);
```

Comment:
No need for second set of square brackets -- useState will use the type provided to generate default -- change to `React.useState<File[]>`